tensorboard/plugins/mesh/mesh_plugin.py

```diff
@@ -258,8 +269,9 @@ def _serve_mesh_metadata(self, request):
 
     # We convert the tensor data to text.
     response = [
-        self._get_tensor_metadata(tensor, meta.content_type, meta.shape,
-                                  meta.json_config)
+        self._get_tensor_metadata(
+          tensor, meta.content_type, meta.components, meta.shape,
```

Comment:
You will need to update doc: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/mesh/http_api.md